src/matlab/IKServer.m

```diff
@@ -174,13 +175,35 @@
     end
 
     function obj = addAffordance(obj, affordanceName)
-      filename = [getenv('DRC_PATH'), '/drake/systems/plants/test/', affordanceName, '.urdf'];
+      filename = [getenv('DRC_PATH'), '/drake/drake/systems/plants/test/', affordanceName, '.urdf'];
       options = struct('floating', false);
       xyz = zeros(3,1);
       rpy = zeros(3,1);
       obj.robot = obj.robot.addRobotFromURDF(filename , xyz, rpy, options);
       obj.robot = compile(obj.robot);
     end
+    
+    function obj = addAffordanceToHand(obj, side, affordance, q, name)
+      hand.left = obj.robot.findLinkId('LeftPalm');
+      hand.right = obj.robot.findLinkId('RightPalm');
```

Comment:
I think this looks like a general function, except for these hard coded link names.